### PR TITLE
Fix crash when no profile applied

### PR DESCRIPTION
**Pull request by llvasconcellos**
The app crashes as described in issue #103. 
Now the client does not crash but displays an empty chart with and no forms. 
This way the user can see clearly that there is something wrong with the profile or it's missing. 

device-2015-10-28-215316

### CODE REVIEW

**Comment by zestyping** [comment]
This is great!  Thanks for fixing this.

File: app/src/main/java/org/projectbuendia/client/sync/ChartDataHelper.java [hunk 1]
```diff
@@ -211,32 +211,37 @@ public Chart getChart(String uuid) {
...
-                            tileGroups.add(tileGroup);
-                            tileGroupsById.put(rowid, tileGroup);
-                            break;
-                        case "GRID_SECTION":
-                            ChartSection rowGroup = new ChartSection(label);
-                            rowGroups.add(rowGroup);
-                            rowGroupsById.put(rowid, rowGroup);
-                            break;
+                    String SectionType = Utils.getString(c, ChartItems.SECTION_TYPE);
+                    if (SectionType != null) {
```
Comment (zestyping) [inline app/src/main/java/org/projectbuendia/client/sync/ChartDataHelper.java:215]: Use lowercase for variable names like "sectionType"

**Comment by capnfabs** [comment]
:+1: Nice work! Thanks for fixing this.

File: app/src/main/java/org/projectbuendia/client/sync/ChartDataHelper.java (hunk 1 above)
Comment (llvasconcellos) [inline app/src/main/java/org/projectbuendia/client/sync/ChartDataHelper.java:215]: You are right. Sometimes I wonder where was my mind when I do somethings like this...

**Pull request merged**